rocrate/vocabs.py

```diff
@@ -19,15 +19,11 @@
 # limitations under the License.
 
 import json
-import pkg_resources
+import importlib.resources
 
 # FIXME: Avoid eager loading?
-RO_CRATE = json.loads(pkg_resources.resource_string(
-    __name__, "data/ro-crate.jsonld"
-))
-SCHEMA = json.loads(pkg_resources.resource_string(
-    __name__, "data/schema.jsonld"
-))
+RO_CRATE = json.loads(importlib.resources.read_text(__name__, "data/ro-crate.jsonld"))
+SCHEMA = json.loads(importlib.resources.read_text(__name__, "data/schema.jsonld"))
```

Comment:
I think I found a way to get this working.

First, we need to change `__name__` to `__package__` (see https://github.com/python/importlib_resources/issues/60 for some explanation). This fixes the `'rocrate.vocabs' is not a package` error.

Then we need to tweak the call to `importlib.resources.read_text` as well, because it doesn't support path separators and it's also deprecated ([see docs](https://docs.python.org/3.11/library/importlib.resources.html#importlib.resources.read_text)). Using the recommendation from those docs gives us:

```suggestion
RO_CRATE = json.loads(
    importlib.resources.files(__package__).joinpath("data/ro-crate.jsonld").read_text()
)
SCHEMA = json.loads(
    importlib.resources.files(__package__).joinpath("data/schema.jsonld").read_text()
)
```

With this change I've been able to successfully run the tests locally in Python 3.11 and 3.12.